### ISSUE TITLE
Synchronization with LDAP does not work. Manually and on a schedule.

### ISSUE THREAD

**Issue by v1k7g85** [goauthentik/authentik#8593]
**Describe the bug**
Synchronization with LDAP does not work. Manually and on a schedule. Nothing happens after pressing the button "Run sync again" I checked for 2 days. Synchronization never started and the state of users/groups was not synchronized with LDAP.

**To Reproduce**
It is not clear under what conditions it can be reproduced.
But in my installation it is:

1. Go to 'Directory -> Federation and Social login'
2. Click on 'Run sync again'
3. Nothing happened

**Expected behavior**
Synchronization should work as expected. Periodically according to a schedule and by manually pressing a button. 

**Screenshots**
![2024-02-20-141325_682x416_scrot](https://github.com/goauthentik/authentik/assets/29337735/1621eb9f-778b-43e2-868a-f3420943b97c)

**Logs**
```
{"event": "Task started", "level": "info", "logger": "authentik.root.celery", "pid": 177171, "task_id": "cdffbadd-e5fe-4e9e-af07-69283162def7", "task_name": "ldap_sync_all", "timestamp": "2024-02-20T10
:23:10.141450"}
{"event": "Task published", "level": "info", "logger": "authentik.root.celery", "pid": 177171, "task_id": "0aeaea3ffea148c5a5491b8112a5a87d", "task_name": "authentik.sources.ldap.tasks.ldap_sync_single
", "timestamp": "2024-02-20T10:23:10.160641"}
{"event": "Task finished", "level": "info", "logger": "authentik.root.celery", "pid": 177171, "state": "SUCCESS", "task_id": "cdffbadde5fe4e9eaf0769283162def7", "task_name": "ldap_sync_all", "timestamp
": "2024-02-20T10:23:10.161794"}
```

**Version and Deployment (please complete the following information):**

-   authentik version: 2023.10.7
-   Deployment: [ helm]

**Comment by lukasruckenstuhl** [comment]
Hey,
I had the same issue once. 
In my case I configured the LDAP credentials wrong. However, it has told me so in the Logs.

**Comment by LukaszC86** [comment]
Hello,
in my case manual syncs are working fine on the latest version 2024.02.1 from docker image, but not the automatic ones. I have configured two LDAP sources in "Federation and Social login", one for the users OU, second for the groups OU. The tasks for ldap_sync (or rather ldap_sync_all?) are not created and run every 2 hours as they should be - after 3 hours the manual tasks are removed and not visible in "System Tasks" dashboard... and there is nothing in worker log regarding the automatic LDAP sync. 🙁

**Comment by v1k7g85** [comment]
@lukasruckenstuhl hi!
I checked the credentials by strarting manual synchronization and it works fine.

**Comment by LukaszC86** [comment]
> If I run it manually it's working. After, I can see in system tasks and in ldap provider that it has been synchronized successfully. But after some time, the system tasks for ldap are gone as the status in ldap provider (back to "no sync yet").

As a (hopefully) temporary workaround I'm running the sync in cron on docker host:

`docker exec authentik-worker-1 ak ldap_sync your_ldap_source_slug`

maybe it will help you too. You won't see the sync status in the dashboard, but the objects will be synced.

**Comment by cardboardpig** [comment]
also seeing this issue, ldap sync jobs have vanished, and I can only get an ldap sync to run by running it manually in the worker container.

**Comment by medir** [comment]
We are under a similar problem.

Installation: Docker version 20.10.17, build 100c701 | docker-compose version 1.29.2 
Authentik version: 2024.2.2 

Problem:
The schedule process it's not running as expected every 2h.
If we force the sincronitzation throught "Directory => Federation and Social Login => button Run sync again" the sincronitzation process works fine.

Following the [Troubleshooting](https://docs.goauthentik.io/docs/troubleshooting/ldap_source) we try to force the sync usign cli:

```bash
docker compose run --rm worker ldap_sync *MY-SLUG-SOURCE* 
```
Looking in the logs I can see that it seems that something is misconfigured?  

```bash
{"domain_url": null, "event": "Source does not exist", "level": "warning", "logger": "authentik.sources.ldap.management.commands.ldap_sync", "pid": 7, "schema_name": "public", "slug": "MY-SLUG-SOURCE", "timestamp": "2024-03-06T12:56:37.025335"}
```

Maybe "domain_url" is the problem? Any advice to fix this parameter?

Thanks in advance,

**Comment by LukaszC86** [comment]
@BeryJu this is still not fixed in the latest version. 🙁

**Comment by cardboardpig** [comment]
This is resolved for me in 2024.4.2 with a fresh installation

**Comment by PentaPaetzold** [comment]
For me, jobs are working with docker / v24.6.3 - but LDAP users are not created from Authentik users. Will open a new Issue.

@v1k7g85 Can we close this one?
